src/VolumeLoader.ts

```diff
@@ -205,25 +201,29 @@ export default class VolumeLoader {
 
     const level0 = await openArray({ store: store, path: imagegroup + "/" + dataset0.path, mode: "r" });
     // full res info
-    const w = level0.meta.shape[4];
-    const h = level0.meta.shape[3];
-    const z = level0.meta.shape[2];
+    // const w = level0.meta.shape[4];
```

Comment:
comment out, not delete?

Reply:
I knew you were going to say that. Agree in principle. Currently this serves as documentation on how to get these dimensions and will almost surely be reinstated later (soon) as we get the viewer handling multiresolution data better.

Reply:
If you knew I was gonna say it, you could have added that comment in the code, and saved us both a trip.  :)